custom_components/tahoma/const.py

```diff
@@ -2,52 +2,61 @@
 
 DOMAIN = "tahoma"
 
+BINARY_SENSOR = "binary_sensor"
+CLIMATE = "climate"
+COVER = "cover"
+LIGHT = "light"
+LOCK = "lock"
+SCENE = "scene"
+SENSOR = "sensor"
+SWITCH = "switch"
+
 SUPPORTED_PLATFORMS = [
-    "binary_sensor",
-    "climate",
-    "cover",
-    "light",
-    "lock",
-    "scene",
-    "sensor",
-    "switch",
+    BINARY_SENSOR,
```

Comment:
These contants already exist. See https://github.com/home-assistant/core/blob/dev/homeassistant/components/binary_sensor/__init__.py#L20 for instance.